contrib/stagit/patches/0001-use-system-prefix.patch

```diff
@@ -0,0 +1,33 @@
+From 508b4ba72464e9845e93434d2752103c5e336e79 Mon Sep 17 00:00:00 2001
+From: ttyyls <contact@behri.org>
+Date: Sun, 18 Feb 2024 15:58:17 +0100
+Subject: [PATCH] use system prefix
+
+---
+ Makefile | 8 ++++----
+ 1 file changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index a037df5..5c1771f 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,12 +4,12 @@ NAME = stagit
+ VERSION = 1.2
+ 
+ # paths
+-PREFIX = /usr/local
+-MANPREFIX = ${PREFIX}/man
++PREFIX ?= /usr
++MANPREFIX = ${PREFIX}/share/man
+ DOCPREFIX = ${PREFIX}/share/doc/${NAME}
+ 
+-LIBGIT_INC = -I/usr/local/include
+-LIBGIT_LIB = -L/usr/local/lib -lgit2
++LIBGIT_INC = -I/usr/include
++LIBGIT_LIB = -L/usr/lib -lgit2
```

Comment:
you can pass all of these with make_build_args already. also for PREFIX the default non-make_use_env overrides it too